cassiopeia/src/main/java/edu/asu/diging/gilesecosystem/cassiopeia/web/EditPropertiesController.java

```diff
@@ -36,12 +39,49 @@
     private ISystemMessageHandler messageHandler;
 
     Map<String, String> ocrTypeMap = new HashMap<>();
+    Map<String, String> langTypeMap = new HashMap<>();
+    String defaultLang = new String();
+
 
     @InitBinder
     protected void initBinder(HttpServletRequest request, ServletRequestDataBinder binder, WebDataBinder validateBinder) {
         validateBinder.addValidators(new SystemConfigValidator());
         ocrTypeMap.put(Properties.OCR_PLAINTEXT, propertyManager.getProperty(Properties.OCR_PLAINTEXT));
         ocrTypeMap.put(Properties.OCR_HOCR, propertyManager.getProperty(Properties.OCR_HOCR));
+        
+        String[] langs = getTessLangs();
+        for(int i=1;i<langs.length;i++) {
+        	langTypeMap.put(langs[i],langs[i]);
+        } 
+        if(langTypeMap.containsKey(Properties.ENGLISH)) {
+        	defaultLang = langTypeMap.get(Properties.ENGLISH);
```

Comment:
and what if not?

Reply:
Currently if the list of languages doesn't have English, it populates the first language in the list such as arabic. Is there anything else I need to change in this case? Kindly let me know...will add the logic accordingly.

Reply:
The logic picks the first language from the list if the lang English is not present.Also empty space is displayed if the language-list is empty.